src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -787,6 +793,28 @@ internal long SetRequestContent(HttpRequestMessage request, Stream contentStream
             return streamContent.Headers.ContentLength.Value;
         }
 
+        /// <summary>
+        /// Sets the ContentLength property of the request and writes the specified content to the request's RequestStream.
+        /// </summary>
+        /// <param name="request">The WebRequest who's content is to be set</param>
+        /// <param name="multipartContent">A MultipartFormDataContent object containing multipart/form-data content.</param>
+        /// <returns>The number of bytes written to the requests RequestStream (and the new value of the request's ContentLength property</returns>
+        /// <remarks>
+        /// Because this function sets the request's ContentLength property and writes content data into the requests's stream,
+        /// it should be called one time maximum on a given request.
+        /// </remarks>
+        internal long SetRequestContent(HttpRequestMessage request, MultipartFormDataContent multipartContent)
+        {
+            if (request == null)
+                throw new ArgumentNullException("request");
+            if (multipartContent == null)
+                throw new ArgumentNullException("multipartContent");
```

Comment:
We should use curly braces:
```c#
if ( ... )
{
    ...
}
```
or single line operator.

Reply:
I can do this, but it will not match the surrounding code.

Reply:
I believe we should use approved patterns for new code.
You can wait a view of other maintainers.

Reply:
I agreed with @iSazonov. The new code is somewhat isolated as it's in its own method, so I think it's fine to use the more preferred style.

Reply:
@daxian-dbw Awesome! format is already corrected in the most recent commit for this PR.